### PR TITLE
net-mgmt/zabbix5-proxy: move to v5

### LINKED ISSUE #1830: Support for Zabbix 5.0 LTS
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
`os-zabbix5-proxy` and `os-zabbix-agent` plugins supporting Zabbix 5.0 LTS.

**Describe alternatives you've considered**
None.

**Additional context**
* [What's New in Zabbix 5.0 LTS](https://www.zabbix.com/whats_new_5_0)

### PR DESCRIPTION
**Pull request by mimugmail**
As with last time we run v4 and v5 in parallel for couple of months.
Closes #1830

### CODE REVIEW

File: net-mgmt/zabbix5-proxy/Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		zabbix5-proxy
+PLUGIN_VERSION=		1.3
+PLUGIN_COMMENT=		Zabbix Proxy enables decentralized monitoring
+PLUGIN_DEPENDS=		zabbix5-proxy
+PLUGIN_CONFLICTS=   zabbix4-proxy
```
Comment (fichtner) [inline net-mgmt/zabbix5-proxy/Makefile:5]: would you mind adding the conflict on the other end as well?

File: net-mgmt/zabbix5-proxy/pkg-descr [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Zabbix is an enterprise-class open source distributed monitoring solution.
+
+Zabbix is software that monitors numerous parameters of a network and the
+health and integrity of servers. Zabbix uses a flexible notification
+mechanism that allows users to configure e-mail based alerts for virtually
+any event. This allows a fast reaction to server problems. Zabbix offers
+excellent reporting and data visualisation features based on the stored
+data. This makes Zabbix ideal for capacity planning.
+
+WWW: http://www.zabbix.com/
```
Comment (fichtner) [inline net-mgmt/zabbix5-proxy/pkg-descr:10]: Maybe they offer https ? :D

File: net-mgmt/zabbix5-proxy/src/opnsense/scripts/OPNsense/Zabbixproxy/setup.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Setup database directory
+mkdir -p /var/db/zabbix
+chown -R zabbix:zabbix /var/db/zabbix
+chmod 755 /var/db/zabbix
+
+# Setup logging
+touch /var/log/zabbix_proxy.log
+chown -R zabbix:zabbix /var/log/zabbix_proxy.log
```
Comment (Starkstromkonsument) [inline net-mgmt/zabbix5-proxy/src/opnsense/scripts/OPNsense/Zabbixproxy/setup.sh:10]: Would you mind moving the logs to `/var/log/zabbix` to have them in the same folder as the agent logs?

**Comment by Starkstromkonsument** [comment]
Thank you for providing this upgrade. I'm really looking forward to upgrade my zabbix environment :-)

File: net-mgmt/zabbix5-proxy/Makefile (hunk 1 above)
Comment (mimugmail) [reply net-mgmt/zabbix5-proxy/Makefile:5]: Hope it's ok when I do this in one PR? :)

File: net-mgmt/zabbix5-proxy/pkg-descr (hunk 2 above)
Comment (mimugmail) [reply net-mgmt/zabbix5-proxy/pkg-descr]: Yep

File: net-mgmt/zabbix5-proxy/src/opnsense/scripts/OPNsense/Zabbixproxy/setup.sh (hunk 3 above)
Comment (mimugmail) [reply net-mgmt/zabbix5-proxy/src/opnsense/scripts/OPNsense/Zabbixproxy/setup.sh]: Sure, why not

**Comment by mimugmail** [comment]
Looks good on my end now ...

File: net-mgmt/zabbix5-proxy/Makefile (hunk 1 above)
Comment (fichtner) [reply net-mgmt/zabbix5-proxy/Makefile:5]: sure, thanks!

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**